argus/util.py

```diff
@@ -54,6 +54,7 @@
 OPEN_NEBULA_SERVICE = 'opennebula'
 CLOUD_STACK_SERVICE = 'cloudstack'
 MAAS_SERVICE = 'maas'
+DIGITALOCEAN_SERVICE = "digitalocean"
 
 DEFAULT_METADATA_URL = {
```

Comment:
You could add the default metadata URL for DO (I think is this one `http://169.254.169.254/metadata/v1/`).